packages/next/build/swc/index.js

```diff
@@ -101,8 +101,11 @@ async function tryLoadWasmWithFallback(attempts) {
     if (!downloadWasmPromise) {
       downloadWasmPromise = downloadWasmSwc(nextVersion, wasmDirectory)
     }
-    await downloadWasmPromise
-    let bindings = await loadWasm(pathToFileURL(wasmDirectory).href)
+    let [, bindings] = await Promise.all([
+      downloadWasmPromise,
+      loadWasm(pathToFileURL(wasmDirectory).href)
+    ])
```

Comment:
No, don't do this here. We are first downloading the wasm, then loading it. These two tasks can not happen in parallel, obviously.